package.json

```diff
@@ -4,7 +4,10 @@
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react-hooks": "^3.2.1",
     "@testing-library/user-event": "^7.1.2",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
     "eslint": "^6.8.0",
```

Comment:
I think these can be dropped down into the devDependencies, along with the @testing items, too. We want to try to keep the dependencies as small as possible for the deployment size. So go through these and move as much as you can into the devDependencies section.